packages/platform-ios/native_modules.rb

```diff
@@ -57,10 +63,10 @@ def use_native_modules!(root = "..", config = nil)
     end
 
     podspec_dir_path = Pathname.new(File.dirname(podspec_path))
-    project_root = Pathname.new(config_root)
+    
     relative_path = podspec_dir_path.relative_path_from project_root
```

Comment:
Now that we don't have custom root, I think instead of using `project_root` we need to use cwd, because we look for – a relative path from cwd (ios dir) to our native module (in node_modules or custom path). This fixes the Slider repo example for me:
```suggestion
    relative_path = podspec_dir_path.relative_path_from Dir.pwd
```